lib/Sema/CSGen.cpp

```diff
@@ -4242,6 +4242,12 @@ namespace {
         return Action::SkipNode(expr);
       }
 
+      // Find all of the entries on the left-hand side of an assignment
+      // expression.
+      if (auto assign = dyn_cast<AssignExpr>(expr)) {
+        recordAssignmentTargets(assign->getDest());
```

Comment:
This looks similar to what happens with argument labels, should probably be moved into `ConstraintGenerator::visitAssignExpr` itself.

Reply:
This code is gone now, since I'm using parent pointers